source/client-side-encryption/client-side-encryption.rst

```diff
@@ -1236,6 +1243,21 @@ EncryptOpts
       algorithm: String,
       contentionFactor: Optional<Int64>,
       queryType: Optional<String>
+      rangeOpts: Optional<RangeOpts>
+   }
+
+   // NOTE: The Range algorithm is experimental only. It is not intended for public use.
```

Comment:
>It is not intended for public use

Can you please clarify what this mean? Is it the same as beta?

Reply:
It is not the same as beta. This comment was requested from product (see [DRIVERS-2286](https://jira.mongodb.org/browse/DRIVERS-2286?focusedCommentId=4975162&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4975162)).